site/assignments/zombies/zombies.rst

```diff
@@ -58,6 +58,24 @@ Part 0 --- Read the Assignment
 Read the assignment description in its entirety before starting.
 
 
+City Data Structure
+-------------------
+
+Every *city* in the simulation is represented as a list ````[name, infected, [neighbours]]``.
+
+* ``name`` is a string with the name of the city
+* ``infected`` is a ``bool`` --- ``True`` if the city is infected with zombies, ``False`` otherwise
+* ``[neighbours]`` is a *list* containing the city numbers (indexes) of the cities connected to this city
+
+For example, a city named ``"City 0`` that is not infected and three ntighbours: cities with indices ``3``, ``6``, and
+``7``, would be represented as ``["City 0", False, [3, 6, 7]]``.
+
+The *world* is made up of multiple cities, which are also stored in a list. Therefore, a world with cities 1, 2, 3, ...
+would be represented as ``[city0, city1, city2, ...]``, where ``city0`` would be a stand in for a whole city data
```

Comment:
Reads odd, 1 based index is used then immediately 0 based is used. Pick one.

Reply:
good catch :nerd_face: